src/theme.ts

```diff
@@ -65,12 +59,12 @@ export const THEME: Theme = {
   backgroundColor: PALETTE.blueTintedGray,
   borderColor: PALETTE.blue,
   collapseBackgroundColor: PALETTE.lightBlue,
-  containerBorderColor: DEPRECATED_PALETTE.gray2,
+  containerBorderColor: PALETTE.gray4,
   dividerColor: PALETTE.gray4,
   focusedRowColor: PALETTE.lightBlue,
   menuGroupBackgroundColor: PALETTE.gray1,
-  panelBackgroundColor: DEPRECATED_PALETTE.gray1,
-  tableBorderColor: PALETTE.gray3,
```

Comment:
Looks better?

Reply:
Since two of the backgrounds are now also using gray3, I decided to make this a different shade - the one used for other borders/dividers.